lib/user-interface/react-app/src/common/types.ts

```diff
@@ -25,6 +25,7 @@ export interface AppConfig {
     rag_enabled: boolean;
     cross_encoders_enabled: boolean;
     sagemaker_embeddings_enabled: boolean;
+    enable_embedding_models_via_sagemaker: boolean;
```

Comment:
What is the difference between the two?

Reply:
Thanks for pointing out, I misinterpreted the other config. Removed it now.